ckanext/knowledgehub/plugin.py

```diff
@@ -20,6 +24,20 @@ def configure(self, config):
         af_db_setup()
         return config
 
+   # IActions
+    def get_actions(self):
+        return {
+            'analytical_framework_delete': knowledgehub_actions.analytical_framework_delete,
+            'analytical_framework_list': knowledgehub_actions.analytical_framework_list,
+        }
+
+    # IAuthFunctions
+    def get_auth_functions(self):
+        return {
+            'analytical_framework_delete': knowledgehub_auth.analytical_framework_delete,
+            'analytical_framework_list': knowledgehub_actions.analytical_framework_list,
```

Comment:
I suppose the auth action should be taken from knowledgehub_auth instead knowledgehub_actions.